tokenizers/src/pre_tokenizers/byte_level.rs

```diff
@@ -510,4 +546,30 @@ mod tests {
             "Hello there dear friend! [PA D]"
         );
     }
+
+    #[test]
+    fn deserialization() {
+        // Before use_regex
+        let byte_level: ByteLevel = serde_json::from_str(
+            r#"{"type": "ByteLevel", "add_prefix_space": true, "trim_offsets": false}"#,
+        )
+        .unwrap();
+        // true
+        assert!(byte_level.use_regex);
+
+        // Loading works, new future BC test.
+        let byte_level: ByteLevel = serde_json::from_str(
+            r#"{"type": "ByteLevel", "add_prefix_space": true, "trim_offsets": false, "use_regex": true}"#,
+        )
+        .unwrap();
+        // true
```

Comment:
idem

Reply:
Resolving to keep discussion in one place.